### DISCUSSION TITLE
ATC_v47 temp and hum offset (pvvx)

### DISCUSSION THREAD

**Discussion by Lassska** [pvvx/ATC_MiThermometer#490] [Q&A] [answered]
why is the offset for temperature and humidity missing in firmware ATC_v47

**Comment by pvvx** [comment, marked as answer]
Since version 4.7, all coefficients for the sensor are available in maximum resolution.
![image](https://github.com/pvvx/ATC_MiThermometer/assets/12629515/36e6f4be-8c80-453c-a783-3ca5bab5e457)
